llvm/include/llvm/Analysis/DXILResource.h

```diff
@@ -441,7 +439,7 @@ ModulePass *createDXILResourceTypeWrapperPassPass();
 //===----------------------------------------------------------------------===//
 
 class DXILBindingMap {
```

Comment:
An open question: Should this also be renamed since its the Map for `ResourceInfo`s not `ResourceBindingInfo`s?

Reply:
Yes, it should be renamed as well.